<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KiraWavTar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>What to do:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="combineWAVRadioButton">
        <property name="text">
         <string>Combine WAV files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="extarctWAVRadioButton">
        <property name="text">
         <string>Extract WAV file</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="optionStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="combineOptionPage">
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Save combined file to:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Target format:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Folder contains WAV files:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="subdirCheckBox">
          <property name="toolTip">
           <string>Make program to scan subfolders (and their subfolders), and combine all WAV files found. The folder structure will be stored in description file, as this will be reconstructed when extract.</string>
          </property>
          <property name="text">
           <string>scan subfolders</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DirNameEditWithBrowse" name="combineDirPathWidget" native="true"/>
        </item>
        <item row="3" column="1">
         <widget class="FileNameEditWithSaveBrowse" name="combineResultPathWidget" native="true"/>
        </item>
        <item row="5" column="0" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="WAVFormatChooserWidget" name="combineWAVFormatWidget" native="true"/>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="extractOptionPage">
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>WAV file to extract:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="extractSelectionCheckBox">
          <property name="text">
           <string>Choose which ones to extract...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Save results to:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QRadioButton" name="extractFormatCustomRadioButton">
            <property name="text">
             <string>Use unified format:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">extractFormatButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="extractFormatSrcRadioButton">
            <property name="text">
             <string>Same as source when combing</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">extractFormatButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="WAVFormatChooserWidget" name="extractFormatCustomChooser" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="DirNameEditWithBrowse" name="extractResultPathWidget" native="true"/>
        </item>
        <item row="5" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="FileNameEditWithBrowse" name="extractSrcPathWidget" native="true"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Output format:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="langButton">
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uiThemeButton">
        <property name="text">
         <string>UI Theme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="leftBottomButtonLine">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirNameEditWithBrowse</class>
   <extends>QWidget</extends>
   <header location="global">kira/widgets/dirnameeditwithbrowse.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileNameEditWithBrowse</class>
   <extends>QWidget</extends>
   <header location="global">kira/widgets/filenameeditwithbrowse.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileNameEditWithSaveBrowse</class>
   <extends>QWidget</extends>
   <header location="global">kira/widgets/filenameeditwithbrowse.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WAVFormatChooserWidget</class>
   <extends>QWidget</extends>
   <header>wavformatchooserwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>extractFormatCustomRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>extractFormatCustomChooser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="extractFormatButtonGroup"/>
  <buttongroup name="modeButtonGroup"/>
 </buttongroups>
</ui>
